community/ec2.tf

```
@@ -2,4 +2,10 @@
 resource "aws_instance" "discourse" {
   ami           = "ami-003634241a8fcdec0"
   instance_type = "m5a.4xlarge"
+  tags = {
+    "Name" = "Discourse"
+  }
+  tags_all = {
+    "Name" = "Discourse"
```

```suggestion
    Name = "Discourse"
```

I thought `tags_all` was generated from tags, is that really needed? it feels a bit verbose to have it here.

---

You're right; I added it because `terraform plan` reported it as change, but without `tags_all`, it says `no change`. So I'll remove it.